src/qmlapplication.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <coreservices.h>
+#include <util/cmdlineargs.h>
+
+#include <QApplication>
+#include <QFileSystemWatcher>
+#include <QQmlApplicationEngine>
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+class QmlApplication : public QObject {
+    Q_OBJECT
+  public:
+    QmlApplication(
+            QApplication* app,
+            std::shared_ptr<CoreServices> pCoreServices,
+            const CmdlineArgs& args);
+    ~QmlApplication() = default;
+
+  public slots:
+    void loadQml(const QString& path);
+
+  private:
+    std::shared_ptr<CoreServices> m_pCoreServices;
+    const CmdlineArgs& m_cmdlineArgs;
```

Comment:
We should not store a plain reference to CmdlineArgs as a member and instead obtain it with CmdlineArgs::Instance(). This is already wrong in CoreServices, I could submit a PR to fix that.

Reply:
CmdlineArgs was unused copypasta from #3448 anyway. Removed and force pushed.